turbo.json

```diff
@@ -124,59 +116,65 @@
       "cache": false
     },
     "lint:attw": {
-      "dependsOn": [],
       "outputs": [],
       "cache": false
     },
     "bundlewatch": {
       "dependsOn": ["build"],
+      "env": [
+        "BUNDLEWATCH_GITHUB_TOKEN",
+        "CI_BRANCH",
+        "CI_BRANCH_BASE",
+        "CI_REPO_NAME",
+        "CI_REPO_OWNER",
+        "CI_COMMIT_SHA"
+      ],
```

Comment:
❓ Does this whitelist these env vars for the task?

Reply:
Yup